### DISCUSSION TITLE
How to integrate Polyflow with Camunda Platform Run

### DISCUSSION THREAD

**Discussion by FlorianCassayre** [holunda-io/camunda-bpm-taskpool#938] [Q&A]
I was wondering if Polyflow could be used together with the [Camunda Platform Run](https://docs.camunda.org/manual/7.20/user-guide/camunda-bpm-run/).
I have not seen anything about it in the documentation, however this is a hard requirement on our side as we are trying to reduce the amount of artifacts that we maintain. Our current platform application consists of a JAR downloaded from the above link along with the various configuration files, and we would like to keep it that way ideally.

Thanks in advance, on behalf of our team at CERN.

**Comment by FlorianCassayre** [comment]
@zambrovski Kindly including you in the loop.

**Reply by zambrovski** [reply]
Interesting question, I have to consider how we can do so. Currently, this means that you need a deployment inside the engine (a JAR, which is the engine collector).

**Reply by FlorianCassayre** [reply]
Thank you for your response.

In the meantime we have come up with a solution involving DB triggers to maintain our own datastructure based on updates of the process instance. This works, even though it may not be very portable/safe; so we are still interested by a alternative potential solution involving a pre-packaged JAR of Polyflow.
